ts_ankle_draft.json

```diff
@@ -0,0 +1,42 @@
+{
+  "information": {
+    "summary": "This file contains validation data for the ankle_draft() function in pythermalcomfort,jsthermalconfort,calcAD() in R, ",
+    "version": "1.0.0",
+    "license": "MIT"
+  },
+  "tolerance": {
+    "ppd": 0.1
```

Comment:
this variable should have the same name as in the outputs

Reply:
sure